src/ci/bin/test.inc.sh

```diff
@@ -1658,37 +1631,6 @@ cromwell::build::check_published_artifacts() {
     fi
 }
 
-cromwell::build::publish_artifacts() {
```

Comment:
This function still appears to be called here: https://github.com/broadinstitute/cromwell/blob/5ba18c74c10bb72e1a3d774dc4bae2ef1ea6020d/src/ci/bin/testSbt.sh#L24

I haven't been following the various build changes, so it's possible that `testSbt.sh` is no longer called in GHA?

Reply:
Correct, we have another way of running all the unit tests now. I'll remove that entire script, too, since it's unused. Good catch.